os/drivers/lcd/lcd_dev.c

```diff
@@ -444,8 +447,10 @@ int lcddev_register(struct lcd_dev_s *dev)
 		lcddbg("ERROR: Failed to allocate memory for LCD flush swap buffer\n");
 		return -ENOMEM;
 	}
-
-	int pid = kernel_thread("LCD Frame flusing", 204, 8192, lcd_flushing_thread, NULL);
+	itoa((int)lcd_info, parm_buf, 16);
+	parm[0] = parm_buf;
```

Comment:
`param[0] = itoa((int)lcd_info, parm_buf, 16);`

And don't we need to memset parm_buf?

Reply:
I don't know why we need to convert it to ascii and to reconvert to integer.
Is it possible to give the address itself without converting?

Reply:
It is not possible to pass the address without converting it to string because kernel_thread() function accept the parameter of type FAR char *const argv[].
**int kernel_thread(FAR const char *name, int priority, int stack_size, main_t entry, FAR char *const argv[])**

Reply:
> `param[0] = itoa((int)lcd_info, parm_buf, 16);`
> 
> And don't we need to memset parm_buf?

I don't think it is needed
could you please explain why you think so?